tutorials/roofit/rf615_learned_likelihoods.py

```diff
@@ -0,0 +1,231 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook
+## Use Simulation Based Inference (SBI) in RooFit
+##
+## This tutorial shows how to use SBI in ROOT. As reference distribution we
+## choose a simple uniform distribution. The target distribution is chosen to
+## be gaussian with different mean values.
+## The classifier is trained to discriminate between the reference and target
+## distribution.
+## We see how the neural networks generalize to unknown mean values.
+##
+## We compare the approach of using the likelihood ratio trick to morphing
+##
+## \macro_code
+## \macro_output
+##
+## \date July 2024
+## \author Robin Syring
+
+
+import ROOT
+import numpy as np
+from sklearn.neural_network import MLPClassifier
+
+# The samples used for training the classifier in this tutorial / rescale for more accuracy
+n_samples = 1000
+
+
+# morphing as a baseline 
+def morphing(setting):
+    # set up a frame for plotting
+    frame1= x_var.frame()
```

Comment:
I would move this after line 66 and comment it out, so it's together with the other plotting code